components/WorkerView/__snapshots__/WorkerSearch.spec.tsx.snap

```diff
@@ -52,24 +52,26 @@ exports[`WorkerSearch should display search results 1`] = `
         </div>
       </div>
     </form>
-    <div>
+    <div
+      class="detailsContainer"
+    >
       <h2
-        class="gov-weight-lighter"
+        class="lbh-heading-h2 gov-weight-lighter title"
       >
         Worker's details found in system
       </h2>
       <p
-        class="govuk-body"
+        class="lbh-body govuk-body"
```

Comment:
you don't need the extra `govuk-body` class for paras. in fact, you might not need any classes at all since i think we're using global styles